tests/award.js

```diff
@@ -1,383 +1,384 @@
-            transactionAdd = "methodToCall.addTransaction.anchorTransactions" + awardNumber + hierarchyNumber
-        })
-        .perform(function(client, done) {
-            client
-            .setValue('input[id="awardHierarchyNodeItems[1].currentFundEffectiveDate"]', '04/01/2014')
-            .setValue('input[id="awardHierarchyNodeItems[1].obligationExpirationDate"]', '04/30/2017')
-            // Adding transactions
-            .click(`input[name="${transactionTabNumber}"]`)
-
-            .click('select[id="transactionBean.newPendingTransaction.sourceAwardNumber"] option[value="000000-00000"]')
-            .click(`select[id="transactionBean.newPendingTransaction.destinationAwardNumber"] option[value="${awardIdAccount}"]`)
-            
-            .clearValue('input[id="transactionBean.newPendingTransaction.obligatedAmount"]')
-            .setValue('input[id="transactionBean.newPendingTransaction.obligatedAmount"]', '100000.00')
-            .clearValue('input[id="transactionBean.newPendingTransaction.anticipatedAmount"]')
-            .setValue('input[id="transactionBean.newPendingTransaction.anticipatedAmount"]', '100000.00')
-            .click(`input[name="${transactionAdd}"]`)
-            .click('input[name="methodToCall.blanketApprove"]')
-            .url(`https://res-tst1.kuali.co/kc-dev/kew/DocHandler.do?command=displayDocSearchView&docId=${documentNumber}`)
-            .pause(1000)
-            .click('input[name="methodToCall.headerTab.headerDispatch.reload.navigateTo.budgets"]')
-            .setValue('input[name="newBudgetVersionName"]', 'Award Budget AFT')
-            .click('input[name="methodToCall.addBudgetVersion"]')
-            .pause(1000)
-            .sendKeys('input[name="methodToCall.openBudgetVersion.line0"]', client.Keys.ENTER)
-            .waitForElementVisible('input[name="methodToCall.headerTab.headerDispatch.save.navigateTo.parameters"]', 3000)
-            .click('input[name="methodToCall.headerTab.headerDispatch.save.navigateTo.parameters"]')
-            .pause(1000)
-            .clearValue('input[id="document.budget.budgetPeriods[0].directCostLimit"]')
-            .setValue('input[id="document.budget.budgetPeriods[0].directCostLimit"]', '25000.00')
-            .clearValue('input[id="document.budget.budgetPeriods[1].directCostLimit"]')
-            .setValue('input[id="document.budget.budgetPeriods[1].directCostLimit"]', '25000.00')
-            .clearValue('input[id="document.budget.budgetPeriods[2].directCostLimit"]')
-            .setValue('input[id="document.budget.budgetPeriods[2].directCostLimit"]', '25000.00')
-            .clearValue('input[id="document.budget.budgetPeriods[3].directCostLimit"]')
-            .setValue('input[id="document.budget.budgetPeriods[3].directCostLimit"]', '25000.00')
-
-            .click('input[name="methodToCall.headerTab.headerDispatch.save.navigateTo.expenses"]')
```

Comment:
These shouldn't be removed. Must have committed some garbage. I'll fix this before merge.